vim-patch:8.1.1824

janlazo opened the pull request.

janlazo commented on src/nvim/testdir/test_spell.vim:
```diff
@@ -303,7 +304,7 @@ func Test_zz_sal_and_addition()
   set spl=Xtest.latin1.spl spell
   call assert_equal('kbltykk', soundfold('goobledygoook'))
-  call assert_equal('kprnfn', soundfold('k�op�r�n�ven'))
+  call assert_equal('kprnfn', soundfold('kóopërÿnôven'))
```
Not part of the patch but I saved the file with `set encoding=utf-8`.

blueyed replied:
Then the added note is not true anymore however - better to keep it like Vim then I guess.

janlazo replied:
Redid it with latin1 encoding in Vim.

janlazo committed:
vim-patch:8.1.1824: crash when correctly spelled word is very long

Problem:    Crash when correctly spelled word is very long. (Ben Kraft)
Solution:   Check word length before copying. (closes vim/vim#4778)
https://github.com/vim/vim/commit/5bcc5a1ff94bbab1b175e35a72e3df974106b393
